rand/rand.go

```diff
@@ -0,0 +1,19 @@
+// Package rand is a convenience wrapper aroung golang rand
+package rand
+
+import (
+	"math/rand"
+	"time"
+)
+
+var r *rand.Rand
+
+// GetRandGenerator seeds a random number generator based on current time
+func GetRandGenerator() *rand.Rand {
```

Comment:
Instead of exposing a `GetRandGenerator` function, we could just expose a `Uint64()` function instead, since that's all we use (and probably will use in the foreseeable future)

Reply:
Upon closer inspection it looks like we do use some other rand functions too, so maybe not worth the effort 🤷

Reply:
Not a bad idea for now: 3309d66a

If we expand the variety of rand functions used, we should probably revert this approach.